.travis.yml

```diff
@@ -2,7 +2,7 @@ language: ruby
 cache: bundler
 install:
 - bundle install
-- npm install
+- yarn
```

Comment:
I think on CI this should be `yarn install --pure-lockfile` (https://github.com/yarnpkg/yarn/issues/570#issuecomment-255091119)

Reply:
will fix

Reply:
might not be necessary with recent versions of yarn